controllers/userC.js

```diff
@@ -23,4 +23,28 @@ const createUser = asyncWrapper(async(req, res, next) => {
     res.status(201).json({ msg: 'User created successfully', user: newUser });
 });
 
-module.exports = { createUser }
\ No newline at end of file
+const userForgotPassword = asyncWrapper(async (req, res, next) => {
+    const user = await fetchUser({ email: req.body.email });
+    if (!user) {
+        return next(createCustomError('User not found', 404));
+    }
+    const token = jwt.sign({
+        id: user._id,
+        email: user.email
+    }, process.env.TOKEN)
+
+    const passwordChangeLink = `${req.protocol}://${req.get("host")}/userss/change_password/${user._id}/${token}`;
+    const message = `Click this link: ${passwordChangeLink} to set a new password`;
+
+    sendEmail({
+        email: user.email,
+        subject: 'Forget password link',
+        message: message
+    });
+
+    res.status(200).json({
+        message: "Email has been sent"
+    });
+});
+
+module.exports = { createUser, userForgotPassword }
```

Comment:
Please, Add an extra line to all your files.

Reply:
Okay, noted